bin/pycbc_multi_inspiral

```diff
@@ -661,7 +661,10 @@ with ctx:
                 ), f'SNR time series for {ifo} is empty'
                 norm_dict[ifo] = norm
                 corr_dict[ifo] = corr.copy()
-                idx[ifo] = ind.copy()
+                idx[ifo] = ind.astype(np.int32)
+                # change to integer indexes because we will have to do
+                # arithmetic with time_delay_idx, which can be positive 
+                #or negative
```

Comment:
The comment usually goes above the line it is referring to. Also, add "signed" before integer to be more clear.